modules/lang-painless/src/main/resources/org/elasticsearch/painless/spi/java.lang.txt

```diff
@@ -729,7 +729,7 @@ class java.lang.StrictMath {
   double nextDown(double)
   double nextUp(double)
   double pow(double,double)
-  double random()
+  double random() @nondeterministic
   double rint(double)
   long round(double)
   double scalb(double,int)
```

Comment:
what about `System#currentTimeMillis` and `System#nanoTime` that also occur in that file. Should we annotate them too?

Reply:
Absolutely, I'm taking another pass at these files now in case there are other misses.